website/src/community/integrations.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: section
+title: "Integrations"
+permalink: /community/integrations/
+section_menu: section-menu/community.html
+redirect_from:
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Integrations
+
+The following external projects integrate with Apache Beam:
+
+## Kettle
```

Comment:
The name of the thing is unclear to me. I would at least in parentheses somewhere say "(aka Pentaho Data Integration)".

Reply:
Indeed, the name does not seem to be clearly defined, so let's mention both.